sdk/metric/doc.go

```diff
@@ -31,6 +31,14 @@
 // is being run on. That way when multiple instances of the code are collected
 // at a single endpoint their origin is decipherable.
 //
+// To avoid leaking memory, the SDK returns the same instrument for calls to
+// create new instruments with the same Unit, Name, and Description.
```

Comment:
nit (use the same order as later)

```suggestion
// create new instruments with the same Name, Unit, and Description.
```